logd/store/badger/store.go

```diff
@@ -45,3 +45,42 @@ func (sto *Store) GetLog(name string) store.Log {
 		name:  name,
 	}
 }
+
+func (sto *Store) NextLog(after store.Log) (store.Log, error) {
+	prefix := []byte{}
+	if after != nil {
+		prefix = append([]byte(after.Name()), 255)
+	}
+	var nextName string
+	if err := sto.db.View(func(txn *badger.Txn) error {
+		opts := badger.DefaultIteratorOptions
+		opts.PrefetchSize = 1000
```

Comment:
This option is not used with `PrefetchValues = false`

Reply:
D'oh. OK.